extensions/amp-analytics/0.1/vendors.js

```diff
@@ -1451,6 +1451,36 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
     },
   },
 
+  'piStats': {
+      'requests': {
+          'host': 'https://events.pi-stats.com',
+          'basePrefix': '${host}/eventsamp/?' +
+          'e=PageLoad&' +
+          'pid=${property}&' +
+          'url=${ampdocUrl}&' +
+          'cnt=${cntId}&' +
+          'lang=${language}&' +
+          'ref=${documentReferrer}&' +
+          'id=${clientId(piStatsDEVICEID)}&'+
+          'ua=${userAgent}&'+
+          'ctype=web&'+
+          'blang=${browserLanguage}&'+
+          'v=2.0&'+
+          'dist=Javascript'
+      },
+      'triggers': {
+          'defaultPageview': {
+              'on': 'visible',
+              'request': 'pageview',
+          },
+      },
+      'transport': {
```

Comment:
just to confirm that you only support transport method image. We suggest using beacon, AMP runtime will handle cases where browser doesn't support beacon and fallback to transport method image.

Reply:
Ok will add beacon as well in that case.

Reply:
Have you made the change? I couldn't find it.